tests/src/Common/test_runtime/project.json

```diff
@@ -2,9 +2,6 @@
   "dependencies": {
     "Microsoft.NETCore.Platforms": "2.0.0-preview1-25210-01",
     "Microsoft.DotNet.CoreCLR.TestDependencies": "1.0.0-prerelease",
-    "jit-dasm": "0.0.1.4",
-    "cijobs": "0.0.1.2",
-    "jit-analyze": "0.0.1.1"
```

Comment:
Because command 
`./Tools/dotnetcli/dotnet restore --packages ./packages /tests/src/Common/test_runtime/project.json`

getting failed otherwise with the following errors

```
error: Package System.CommandLine 1.0.0.1 is not compatible with netcoreapp2.0 (.NETCoreApp,Version=v2.0). Package System.CommandLine 1.0.0.1 supports: net35 (.NETFramework,Version=v3.5)
error: Package TypeUtils 1.0.0.2 is not compatible with netcoreapp2.0 (.NETCoreApp,Version=v2.0). Package TypeUtils 1.0.0.2 supports: net35 (.NETFramework,Version=v3.5)
error: One or more packages are incompatible with .NETCoreApp,Version=v2.0.
...
error: Package System.CommandLine 1.0.0.1 is not compatible with netcoreapp2.0 (.NETCoreApp,Version=v2.0) / win7-x86. Package System.CommandLine 1.0.0.1 supports: net35 (.NETFramework,Version=v3.5)
error: Package TypeUtils 1.0.0.2 is not compatible with netcoreapp2.0 (.NETCoreApp,Version=v2.0) / win7-x86. Package TypeUtils 1.0.0.2 supports: net35 (.NETFramework,Version=v3.5)
error: One or more packages are incompatible with .NETCoreApp,Version=v2.0 (win7-x86).
log  : Lock file has not changed. Skipping lock file write. Path: /home/rartemev/git/coreclr/tests/src/Common/test_runtime/project.lock.json
log  : /home/rartemev/git/coreclr/tests/src/Common/test_runtime/project.json
log  : Restore failed in 6631ms.
```

If someone has an idea how can I fix it I would be like to know